pkg/security/secl/model/unmarshallers.go

```diff
@@ -72,7 +72,13 @@ func (e *Event) UnmarshalBinary(data []byte) (int, error) {
 	}
 
 	e.TimestampRaw = ByteOrder.Uint64(data[8:16])
-	e.Type = ByteOrder.Uint64(data[16:24])
+	e.Type = ByteOrder.Uint32(data[16:20])
+	if data[20] != 0 {
```

Comment:
Are we sure the bitfield will actually be at the start ? (Like in a big-endian situation)

Reply:
Yes you're right, I removed the bitfield. Thanks